app/models/current_claim.rb

```diff
@@ -61,7 +71,30 @@ def ineligible?
     claims.all? { |c| c.eligibility.ineligible? }
   end
 
+  def eligible_now?
+    claims.any? { |c| c.eligibility.eligible_now? }
+  end
+
+  def eligible_later?
+    claims.any? { |c| c.eligibility.eligible_later? }
+  end
+
   def editable_attributes
     claims.flat_map { |c| c.eligibility.class::EDITABLE_ATTRIBUTES }.uniq
   end
+
+  def eligible_now
+    claims.select { |c| c.eligibility.eligible_now? }
+  end
+
+  def eligible_now_and_sorted
+    eligible_now.group_by { |c| c.award_amount.to_i }.map { |k, claims| claims.sort_by { |c| c.policy.short_name } }.reverse.flatten
```

Comment:
Isn't it meant to be sorted by decreasing monetary amount, so the highest award appears first?

Reply:
That's correct, this is asserted in the specs.

Reply:
I feel like this line does too much, maybe assigning some variables would mke it a bit clearer.